src/python/pants/engine/legacy/graph.py

```diff
@@ -714,6 +715,8 @@ def _eager_fileset_with_spec(
     PathGlobs(
       include=(fs_spec.glob for fs_spec in filesystem_specs),
       glob_match_error_behavior=glob_match_error_behavior,
```

Comment:
So, while we're here: this will be the "global" `GlobMatchErrorBehavior`, which is target-specific, and computed from this option: https://github.com/pantsbuild/pants/blob/71843e7549b68998308dc301dbdfe61b3067198a/src/python/pants/option/global_options.py#L287-L290

It's likely that we always want to explode for unmatched globs on the CLI...? So explicitly passing "error" here.